.github/workflows/test.yml

```diff
@@ -6,10 +6,15 @@ on: [pull_request]
 jobs:
   fast-tests:
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
     timeout-minutes: 15
     strategy:
       matrix:
         python-version: ['3.7', '3.8']
+        experimental: [false]
+        include:
+          - python-version: 3.9
+            experimental: true
```

Comment:
Why are `python-version` and `experimental` lists above, but scalar values here?

Reply:
Oh, hadn't noticed it. Both appear to work fine. But let me edit that commit to match the other values :+1:

Reply:
Done, just need to wait for CI to run and confirm it works too.

Reply:
(I see the GH Actions docs does it the way your originally did it, so maybe it works either way :crossed_fingers: )

Reply:
Bugger:
```
Error: .github/workflows/test.yml (Line: 25, Col: 27): A sequence was not expected
Error: The template is not valid. .github/workflows/test.yml (Line: 25, Col: 27): A sequence was not expected
```

Reply:
But good to know! I guess if you have multiple versions, then you will have to do something like 

```
- python-version: 1
  experimental: true
- python-version: 2
  experimental: true
...
```

Reply:
What does the "experimental" mean exactly?

Reply:
It's just a feature in GitHub actions that allows to mark a branch of the matrix-tree as not-ready I guess? It's really common in Java and JavaScript, for new versions of the JVM or of Node (which are experimental in relation to the code; meaning that they may not work, and the developers/CI are "experimenting" with it? :smile: )

Reply:
OK, fair enough.  Just metadata then, no real consequences :+1:

Reply:
Can remove the experimental thing now 3.9 is out.